css/helpers/flexbox.css

```diff
@@ -39,19 +39,19 @@
 
 /* flex gaps */
 .gap-05 {
-  gap: var(--size-000);
+  gap: var(--scale-05);
 }
```

Comment:
for the classname on line 41, using `05` would confuse someone as `5`
Suggest to use some variable instead , like `xs` or `sm`

Reply:
hey, it's a scale table defined in variables and I'll make a naming convention page in docs to better understand. 
As of now, you can refer to` reset.css` to check on this.

Reply:
Understood, we discussed and you mentioned that you will refactor it lateron, 
we can close this discussion 👍